Baseline glossary entry: mention accessibility and other considerations (#33515)
mdn/content: files/en-us/glossary/baseline/compatibility/index.md

## Extra considerations

> Baseline is a summary guide to browser support, to help you make decisions before committing to a feature.

Reviewer: Minor reword suggestion for clarity

```suggestion
Baseline is a summary guide to browser support to help you make decisions before committing to use a feature in your projects.
```

> But it's not a substitute for testing your application for accessibility, usability, performance, and other considerations.

Reviewer: Another small edit to consider:

```suggestion
Baseline is not a substitute for testing your application for accessibility, usability, performance, security, or other concerns.
```

Baseline may not tell you whether a feature works with:

- Older devices and browser releases

> - Browsers not covered by the Baseline browser set, such as operating system web views

Reviewer: Can we link to the "Baseline browser set"?

Author: I thought about this, but wasn't sure if it was weird to link to the immediately preceding section of the same page. Happy to do it if that's conventional. Something like this?

```suggestion
- Browsers not covered by the Baseline [browser set](#browser_set), such as operating system web views
```

Reviewer: Good point, we're right under the browser set section, so we can omit this 👍🏻

- Assistive technology, such as screen readers

> If Baseline doesn't cover your situation, then consider reading browser compatibility tables on MDN and visiting [caniuse.com](https://caniuse.com/) and [a11ysupport.io](https://a11ysupport.io/).

Reviewer: Copy edit:

```suggestion
If Baseline doesn't cover your situation, you can find more information in the browser compatibility tables on MDN, [caniuse.com](https://caniuse.com/), and [a11ysupport.io](https://a11ysupport.io/).
```
